internal/strs/strs.go

```diff
@@ -133,20 +104,10 @@ func ToSnakeCase(s string) string {
 	return output
 }
 
-// DedupeSlice returns s with no duplicates and no empty strings, in the same order.
+// Dedupe returns s with no duplicates and no empty strings, sorted.
```

Comment:
nit: I think this should still be `DedupeSort` as a form of self-documentation. If the function is just named `Dedupe`, and I input `[]{"b", "a", "c"}`, my natural instinct would be to expect to get the same slice value back. People may miss the Godoc.

Reply:
Good call! Will update.